### PR TITLE
Enable onGetDataClick, skip cache when inserting, other small changes

### PR DESCRIPTION
**Pull request by Nirmay15**

### CODE REVIEW

File: packages/slides/src/server/Code.ts [hunk 1]
```diff
@@ -246,7 +246,7 @@ function serializeBlob(blob) {
   return Utilities.base64Encode(blob.getBytes());
 }
 
-function getImages(links, skipCache = false) {
```
Comment (ashubham) [inline packages/slides/src/server/Code.ts:249]: Generally optional bool arga should default to false. So might be good to change this to useCache.

But that can be another CL

**Review by ashubham — APPROVED** [review]

File: packages/slides/src/server/Code.ts (hunk 1 above)
Comment (Nirmay15) [reply packages/slides/src/server/Code.ts:249]: Ack, will do this in next PR
